packages/mux-player/src/index.ts

```diff
@@ -733,20 +735,41 @@ class MuxPlayerElement extends VideoApiElement {
   }
 
   /**
-   * Return the storyboard URL when a playback ID is provided,
+   * Return the storyboard-src attribute URL
+   */
+  get storyboardSrc() {
+    return this.getAttribute(PlayerAttributes.STORYBOARD_SRC) ?? '';
```

Comment:
question(blocking): Should this instead yield `undefined` by default, per your docs? if so, refactor to e.g.
```js
return this.getAttribute(PlayerAttributes.STORYBOARD_SRC) ?? undefined;
```